### ISSUE TITLE
Login button not accessible

### ISSUE THREAD

**Issue by phipier** [jessisena/TreeCheckerApp#8]
The Login button seems to disappear after a failed attempt to log in. We have to restart the app to log in again.

**Comment by jessisena** [comment]
I can't reproduce the error.  I've tryed using an incorrect username, and incorrect password, and in both cases the button never dissapears. With which kind of error is related your failed attempt to login?

**Comment by phipier** [comment]
I think it happened when I switched to our internal wifi (through which
connection cannot be established apparently)


2018-06-20 15:42 GMT+02:00 Jessica Sena <notifications@github.com>:

**Comment by jessisena** [comment]
Yes, it could be that through your internal wifi you do not have access to the server. That would explain why the button disappears. The app detects connection in the device, but after the first attempt sees that with the available network it does not have real access, so it does not let you login again.

Could you check it with another network and tell me if you still have problems?
